app/controllers/concerns/notifications/validateable.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Notifications
+  module Validateable
```

Comment:
The following comment is for posterity:

There was discussion on alternative approaches to validating the presence of subjects/statuses in the model-level enum, including potentially [overriding `assert_valid_value`]( https://github.com/rails/rails/issues/13971#issuecomment-383905807) in `initializers/active_record_enum.rb` as such:

```rb
module ActiveRecord
  module Enum
    class EnumType < Type::Value
      def assert_valid_value(value)
        unless value.blank? || mapping.has_key?(value) || mapping.has_value?(value)
          nil
        end
      end
    end
  end
end
``` 

Which would allow for us to validate the enum as follow:

```rb
class User < ApplicationRecord
  enum gender: [:male, :female]
  validates :gender, inclusion: { in: genders.keys, message: :invalid }
end
```


@hpjaj and I agreed that while this may be a cleaner way of *validating* it might cause undue developer confusion in the future, and including a concern is a more explicit way of showing a path through the code.